packages/core/database/lib/schema/builder.js

```diff
@@ -316,7 +316,7 @@ const createHelpers = (db) => {
         debug(`Creating column ${addedColumn.name}`);
 
         if (addedColumn.type === 'increments' && !db.dialect.canAddIncrements()) {
-          tableBuilder.integer(addedColumn.name).unsigned().notNullable();
+          tableBuilder.integer(addedColumn.name).unsigned();
```

Comment:
Do we need to check `db.dialect.supportsUnsigned()` here?

Reply:
This can only happen on sqlite because it's the only that cannot add Increment columns but that's a good comment we can add for completeness afterwards we release this fix I don't want to introduce another pbl by mistake :)